src/config.rs

```diff
@@ -425,29 +425,79 @@ pub struct HtmlConfig {
     pub livereload_url: Option<String>,
     /// Should section labels be rendered?
     pub no_section_label: bool,
+    /// Search settings. If `None`, the default will be used.
+    pub search: Option<Search>,
 }
 
 /// Configuration for tweaking how the the HTML renderer handles the playpen.
 #[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
 #[serde(default, rename_all = "kebab-case")]
 pub struct Playpen {
-    /// The path to the editor to use. Defaults to the [Ace Editor].
-    /// 
-    /// [Ace Editor]: https://ace.c9.io/
-    pub editor: PathBuf,
-    /// Should playpen snippets be editable? Defaults to `false`.
+    /// Should playpen snippets be editable? Default: `false`.
     pub editable: bool,
+    /// Copy JavaScript files for the editor to the output directory?
+    /// Default: `true`.
+    pub copy_js: bool,
```

Comment:
This probably belongs in its own PR. Changing how the playpen works and whether we copy across JavaScript files is a non-trivial change.

Reply:
I suppose. Should I revert the changes to `editor`, but leave the `output.html.search.copy-js` setting?

Reply:
Mmm, don't worry about it for now, it's probably not worth the effort. As long as it's noted in the PR description it should be fine.